.github/workflows/labeler-community.yml

```diff
@@ -0,0 +1,12 @@
+# https://github.com/actions/labeler
+
+name: Community PR Labels
+on:
+- pull_request_target
```

Comment:
Is there a set of branches you expect would be the only ones that should be targeted by external PRs? (typically, `master`). If yes we could set them here so that external contributors will not be able to trigger this workflow with PRs targetting weird branches.

See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request-1

Reply:
Good idea, I'll restrict the workflow to only the `master` branch.